configs/sundancedsp_polarberry_defconfig

```diff
@@ -0,0 +1,56 @@
+BR2_riscv=y
+BR2_riscv_custom=y
+BR2_RISCV_ISA_CUSTOM_RVM=y
+BR2_RISCV_ISA_CUSTOM_RVA=y
+BR2_RISCV_ISA_CUSTOM_RVF=y
+BR2_RISCV_ISA_CUSTOM_RVD=y
+BR2_RISCV_ISA_CUSTOM_RVC=y
+BR2_KERNEL_HEADERS_5_15=y
+BR2_BINUTILS_VERSION_2_39_X=y
+BR2_GCC_VERSION_11_X=y
+BR2_TOOLCHAIN_BUILDROOT_CXX=y
+BR2_PACKAGE_HOST_GDB=y
+BR2_SHARED_STATIC_LIBS=y
+BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_MCHP_PATH)/board/microchip/polarberry/patches/"
+BR2_TARGET_GENERIC_HOSTNAME="POLARBERRY"
+BR2_TARGET_GENERIC_ISSUE="Welcome to POLARBERRY Linux"
+BR2_TARGET_GENERIC_ROOT_PASSWD="root"
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_MCHP_PATH)/board/microchip/polarberry/rootfs_overlay"
+BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_MCHP_PATH)/board/microchip/polarberry/post-image.sh"
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_GIT=y
+BR2_LINUX_KERNEL_CUSTOM_REPO_URL="https://github.com/linux4microchip/linux.git"
+BR2_LINUX_KERNEL_CUSTOM_REPO_VERSION="linux-5.15-mchp"
+BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
+BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_MCHP_PATH)/board/microchip/polarberry/linux/linux_polarberry_defconfig"
+BR2_LINUX_KERNEL_DTS_SUPPORT=y
+BR2_LINUX_KERNEL_CUSTOM_DTS_PATH="$(BR2_EXTERNAL_MCHP_PATH)/board/microchip/polarberry/linux/linux_polarberry.dts"
+BR2_PACKAGE_LINUX_TOOLS_GPIO=y
+BR2_PACKAGE_SPIDEV_TEST=y
+BR2_PACKAGE_SPI_TOOLS=y
+BR2_PACKAGE_LIBSYSFS=y
+BR2_PACKAGE_JPEG_TURBO=y
+BR2_PACKAGE_LIBSVG_CAIRO=y
+BR2_PACKAGE_DROPBEAR=y
```

Comment:
Please ensure this, and the entire Buildroot build, is tested and working with Buildroot version 2023.02.